src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  height: 406px;
+  width: 198px;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), fixed values for height and width should be avoided where possible. Consider using max-width and max-height for better responsiveness.